Person.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+#include "validation.h"
+
+#include <iostream>
+#include <string>
+
+using namespace std;
+
+class Person
+{
+private:
+    int ID;
+    string Name;
+    string Password;
+
+public:
+    Person(int ID, string Name, string Password)
+    {
```

Comment:
You should call the setters in the constructor to apply any validation present in them if you created a new object